cms/static/sass/_variables.scss

```diff
@@ -209,12 +210,16 @@ $ui-link-color: $blue-u2;
 $ui-link-color-focus: $blue-s1;
 
 // +Specific UI
-// ==================== 
+// ====================
 $ui-notification-height: ($baseline*10);
 $ui-update-color: $blue-l4;
 
-// +Deprecated 
-// ==================== 
+// +Paths
+// ====================
+$static-path: '..' !default;
```

Comment:
Good abstraction here. In the UXPL we're setting specific paths for certain assets (e.g. fonts) - https://github.com/edx/ux-pattern-library/blob/master/_src/pattern-library/sass/utilities/_variables.scss#L147.

While its true for us that we keep our images, fonts, and other static assets in the same parent dir, there may be some value in creating a few path variables here (in case theme creators want to organize their static assets differently). What do you think about creating at least ``$fonts-path`` and ``$images-path``?